manifests/plugin/filter/match.pp

```diff
@@ -1,18 +1,15 @@
 # https://collectd.org/wiki/index.php/Chains
 define collectd::plugin::filter::match (
-  $chain,
-  $rule,
-  $plugin,
-  $options = undef,
+  String $chain,
+  String $rule,
+  # Same as listed in $collectd::plugin::filter::plugin_matches
+  Enum['regex', 'timediff', 'value', 'empty_counter', 'hashed'] $plugin,
```

Comment:
should we create a custom datatype for this?

Reply:
Yes